f5-distributed-cloud/manifest.json

```
@@ -0,0 +1,53 @@
+{
+   "manifest_version": "2.0.0",
+    "app_uuid": "74c33838-0310-4ef3-95db-c378aece9d8b",
+    "app_id": "f5-distributed-cloud-services",
+    "display_on_public_website": true,
+    "tile": {
+      "overview": "README.md#Overview",
+      "configuration": "README.md#Setup",
+      "support": "README.md#Support",
+      "changelog": "CHANGELOG.md",
+      "description": "Stream and visualize F5 Distributed Cloud Services event logs.",
+      "title": "F5 Distributed Cloud Services",
+      "media": [
+        {
+          "media_type": "video",
+          "caption": "Configuring remote logging from F5 Distributed Cloud Services to Datadog",
+          "image_url": "https://youtu.be/oONsA-k7n5s"
+        },
+        {
+          "media_type": "image",
+          "caption": "The F5 Distributed Cloud Services global log receiver",
+          "image_url": "images/logreceiver-config.png"
+        },
+        {
+          "media_type": "image",
+          "caption": "The F5 Distributed Cloud Services access log overview dashboard.",
+          "image_url": "images/dashboard_image.png"
+        }
+      ],
+      "classifier_tags": [
+        "Offering::Integration",
+        "Supported OS::Linux",
+        "Supported OS::Mac OS",
+        "Supported OS::Windows",
+        "Category::Notification"
+      ]
+    },
+    "author": {
+      "name": "F5 Distributed Cloud Services",
+      "homepage": "https://www.f5.com/cloud",
+      "sales_email": "sales@f5.com",
+      "support_email": "g.coward@f5.com"
+    },
+    "oauth": {},
+    "assets": {
+      "dashboards": {
+        "f5xc_access": "assets/dashboards/f5xc_access.json"
+      },
+      "saved_views": {
+        "f5xc_all": "assets/saved_views/all.json"
+      }
+    }
```

```suggestion
    "assets": {
    },
      "dashboards": {
        "f5xc_access": "assets/dashboards/f5xc_access.json"
      },
      "saved_views": {
        "f5xc_all": "assets/saved_views/all.json"
      }
```
This should fix the error that's appearing in the manifest